guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -3,16 +3,42 @@
 
 In {Project}, you can use the package dependency resolution feature to ensure that any dependencies that packages have within a Content View are added to the dependent repository as part of the Content View publication process.
 
-You can select to resolve package dependencies for any Content View that you want, or you can change the default setting to enable or disable resolving package dependencies for all new Content Views.
+You can select to resolve package dependencies per Content View or have it enabled by default via Settings -> Content -> "content view Dependency Solving Default".
+Enabling content view dependency solving by default is unrecommended unless the pitfalls (discussed later) are well-understood.
 
 Note that resolving package dependencies can cause significant delays to Content View promotion.
-The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View or solve dependencies across repositories.
+The more content and repositories in the content view, the greater the delay.
+
+The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View.
+See {ContentManagementDocURL}Limitations_to_Repository_Dependency_Resolution[Limitations to Repository Dependency Resolution] for more information about when dependency resolution may fail.
 
 .Resolving Package Dependencies and Filters
 
-Filters do not resolve any dependencies of the packages listed in the filters.
+Filters, by default, do not resolve any dependencies of the packages listed in the filters.
 This might require some level of testing to determine what dependencies are required.
 
 If you add a filter that excludes some packages that are required and the Content View has dependency resolution enabled, {Project} ignores the rules you create in your filter in favor of resolving the package dependency.
 
-If you create a content filter for security purposes, to resolve a package dependency, {Project} can add packages that you might consider insecure.
+If you create a content filter for security purposes but choose to resolve dependencies, {Project} can add packages that you might consider insecure.
+
+.Filtering and Dependency Solving Recommendations
+
+Be mindful when combining exclusion filters with dependency solving.
```

Comment:
```suggestion
Be mindful when combining filters with dependency solving.
```

Isn't an include filter really an "exclude everything except" filter?

Reply:
I was thinking more conceptually. Dependency solving + exclusion filters can mean the exclusions will not be respected.